modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/AbstractCSharpCodegen.java

```diff
@@ -586,6 +586,7 @@ public Map<String, ModelsMap> postProcessAllModels(Map<String, ModelsMap> objs)
                 if (allOf != null) {
                     for (CodegenProperty property : allOf) {
                         property.name = patchPropertyName(model, camelize(property.baseType));
+                        property.isNullable = property.isNullable || model.isNullable;
```

Comment:
so if a model has `nullable` set to true, then all its properties are also `nullable` (nullable: true)?

Reply:
Removed this change from the abstract. Since we know anyof and oneof will always be nullable, for value types we can just always put .Value. That means I can just remove the nullable check from the template.